cclib/parser/molcasparser.py

```diff
@@ -656,7 +663,18 @@ def extract(self, inputfile, line):
                             if float(i) != 0:
                                 homos += 1
 
-                    line = next(inputfile)
+                    aonames = []
+                    if line.split():
+                        if line.split()[0] == '1':
+                            while line.split() != [] and line.split()[0] != '--':
+                                aonames.append("%s_%s" % (line.split()[1], line.split()[2]))
+                                info = line.split()[3:]
+                                j = 0
+                                for i in orbital_index:
+                                    mocoeffs[int(i)-1].append(info[j])
```

Comment:
Don't you need to turn those strings into floats somewhere?

Reply:
Looks like this happens in `ccData.arrayify`.

Reply:
@ATenderholt I will make them into floats here instead of leaving that to `ccData.arrayify`. Just seems inconsistent.